specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProperty.json

```diff
@@ -65,182 +95,553 @@
           "readOnly": true,
+          "x-ms-enum": {
+            "name": "BillingProfileStatusReasonCode",
+            "modelAsString": true
+          }
+        },
+        "billingProfilePaymentMethodType": {
+          "description": "The payment method type of the primary payment method for the billing profile.",
+          "enum": [
+            "Unknown",
+            "Credits",
+            "Creditcard",
+            "Checkwire"
+          ],
           "type": "string",
+          "readOnly": true,
           "x-ms-enum": {
-            "modelAsString": true,
-            "name": "BillingProfileStatusReasonCode"
+            "name": "PaymentMethodType",
+            "modelAsString": true
           }
         },
-        "billingTenantId": {
+        "primaryBillingTenantId": {
           "description": "The Azure AD tenant ID of the billing account for the subscription.",
-          "readOnly": true,
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "costCenter": {
-          "description": "The cost center applied to the subscription.",
+          "description": "The cost center applied to the subscription. Available for agreement type Microsoft Customer Agreement and Microsoft Partner Agreement.",
           "type": "string"
         },
+        "customerDisplayName": {
+          "description": "The name of the customer.",
+          "type": "string",
+          "readOnly": true
+        },
+        "customerId": {
+          "description": "The fully qualified ID that uniquely identifies a customer.",
+          "type": "string",
+          "readOnly": true
+        },
         "invoiceSectionDisplayName": {
-          "description": "The name of the invoice section to which the subscription is billed.",
-          "readOnly": true,
-          "type": "string"
+          "description": "The name of the invoice section.",
+          "type": "string",
+          "readOnly": true
         },
         "invoiceSectionId": {
-          "description": "The ID of the invoice section to which the subscription is billed.",
+          "description": "The fully qualified ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionStatus": {
+          "description": "Identifies the status of an invoice section.",
+          "enum": [
+            "Unknown",
+            "Active",
+            "Deleted",
+            "Disabled",
+            "UnderReview",
+            "Warned",
+            "Restricted"
+          ],
+          "type": "string",
           "readOnly": true,
-          "type": "string"
+          "x-ms-enum": {
+            "name": "InvoiceSectionStatus",
+            "modelAsString": true
+          }
+        },
+        "invoiceSectionStatusReasonCode": {
+          "description": "Reason for the specified invoice section status.",
+          "enum": [
+            "PastDue",
```

Comment:
These aren't really invoice section level states, are they? Aren't these billing profile states?

Reply:
@ramkd12 @amberbhargava to comment here